### DISCUSSION TITLE
Add price feed for SKL/USD to Razor Network

### DISCUSSION THREAD

**Discussion by yohanelly95** [razor-network/governance#17] [Ideas]
## Objective
Proposal for a new price feed for **SKL/USD**

### Description

Datasources Pull Request - https://github.com/razor-network/datasources/pull/75

**Comment by manuelbarbas** [comment]
**Title:**

Proposal to Introduce SKL/USD Price Feed to Razor Network Protocol

**Description:**

SKALE is a decentralized modular cloud network designed to bring scalability to Ethereum. By offering high-speed consensus and a zero gas fee network, SKALE aims to make decentralized applications (dApps) more performant and cost-effective. Its token, SKL, plays a pivotal role in securing and incentivising the network. 

Recognizing the growing importance of SKALE and its influence in the DeFi sector, a reliable and decentralized SKL/USD price feed becomes increasingly relevant for a multitude of applications and platforms within the SKALE and Ethereum ecosystem.

- Provide decentralized and trustworthy price data for SKL/USD.
- Cater to a broader spectrum of DeFi projects and platforms requiring this data.
